lte/protos/pipelined.proto

```diff
@@ -276,3 +276,289 @@ service Pipelined {
   // and name
   rpc GetAllTableAssignments (magma.orc8r.Void) returns (AllTableAssignments);
 }
+
+// SMF to UPF
+
+message NodeID {
+    enum NodeIDType {
+         NONE = 0;
+         IPv4 = 1;
+         IPv6 = 2;
+         FQDN = 3;
+    }
+
+    NodeIDType nodetype = 1;
+    string node_id = 2;
+}
+
+//CPFseid message
+message CpFSeid {
+  uint64 f_seid = 1;
+  NodeID node_id = 2;
+}
+
+message OuterHeadRemoval {                //conditional PDR Message Structures
+  uint32 o_hd_remo_descr= 1;
+}
+
+message LocalFTeid {                      //not sure about the message structure
+ uint32 teid = 1;
+ string ipv4_adr = 2; //doubt
+}
+
+message SdfFilters {                               //8.2.5
+  FlowDescriptor fd = 1;
+  bytes ttc = 3;
+  bytes spi = 4;
+  bytes fl = 5;
+  uint32 sdf_id = 6;
+}
+
+message FlowDescriptor {
+FlowMatchSMF match = 1;
+  enum Action {
+    PERMIT = 0;
+    DENY = 1;
+  }
+  Action action = 2;
+}
+
+message FlowMatchSMF {
+  string ipv4_src = 1;
```

Comment:
Lets use 
```
message IPAddress {
  enum IPVersion {
    IPV4 = 0;
    IPV6 = 1;
  }
  IPVersion version  = 1;
  bytes address = 2; // network byte order
}
```
For all ip addresses. This way we can avoid the future refactor on ipv6 change.

This IPAddress proto is in mobilityd.proto (I'm working on a PR that will make this struct available in pipelined.yml